core/src/main/scala/org/apache/spark/ui/jobs/ExecutorTable.scala

```diff
@@ -102,7 +102,7 @@ private[ui] class ExecutorTable(stageId: Int, stageAttemptId: Int, parent: Stage
 
     listener.stageIdToData.get((stageId, stageAttemptId)) match {
       case Some(stageData: StageUIData) =>
-        stageData.executorSummary.toSeq.sortBy(_._1).map { case (k, v) =>
+        stageData.executorSummary.toSeq.sortBy(_._1.toLong).map { case (k, v) =>
```

Comment:
I just checked the usage of sorttable (from kryogenix) used in Spark UI, and actually, we have to sort server side.
Basically, using sorttable js at page loading is wrong. Here's the extract for the sorttable FAQ:

"
Sorting the table when the page is loaded

Lots of people ask, "how do I make sorttable sort the table the first time the page is loaded?" The answer is: you don't. Sorttable is about changing the HTML that is served from your server without a page refresh. When the page is first served from the server, you have to incur the wait for it to be served anyway. So, if you want the table sorted when a page is first displayed, serve the table in sorted order. Tables often come out of a database; get the data from the database in a sorted order with an ORDER BY clause in your SQL. Any solution which involves you running sorttable as soon as the page loads (i.e., without user input) is a wrong solution.
"

So, according to @JoshRosen (good catch Josh ;)), executor ID is not necessary a long (for instance, "driver" executor). Anyway, let me check if I can sort (alphanumeric) on server side, matching the sorttable sort ordering.